src/Defra.Trade.Events.IDCOMS.PLNotifier/Defra.Trade.Events.IDCOMS.PLNotifier.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Defra.Trade.Common" Version="3.0.0" />
+    <PackageReference Include="Defra.Trade.Common.Logging" Version="2.0.13" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="5.16.1" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.4.0" />
```

Comment:
Should we be adding as Isolated instead of in-process?

Reply:
addressed in 0d5d15d

Reply:
Agreed to approach isolated process with wider team